Hyper/PlayerData/InventorySystem/Items/Pickaxes/Pickaxe.cs

```diff
@@ -45,13 +39,7 @@ public override void Down()
         Radius = Math.Max(Radius - 1, 1);
     }
 
-    private enum ModificationType
-    {
-        Mining,
-        Building
-    }
-
-    private void ModifyTerrain(Scene scene, IChunkWorker chunkWorker, float time, Func<Chunk, ModificationFunc> modifier, ref float modificationTime)
+    private void ModifyTerrain(Scene scene, IChunkWorker chunkWorker, float time, ModificationType modificationType, ref float modificationTime)
```

Comment:
why ref for modificationTime

Reply:
this was introduced in some previous commit, but modificationTime is a reference to either _buildTime or _mineTime & is updated in line 85. Just eliminating ifs

Reply:
ok